flow/envs/base.py

```diff
@@ -426,6 +429,13 @@ def reset(self):
         # reset the time counter
         self.time_counter = 0
 
+        # Now that we've passed the possibly fake init steps some rl libraries
+        # do, we can feel free to actually render things
+        if self.should_render:
+            self.sim_params.render = True
+            # got to restart the simulation to make it actually display anything
+            self.restart_simulation(self.sim_params)
```

Comment:
@eugenevinitsky line 437 actually creates an extra restart_simulation (at least when I'm running bayesian_1_env.py)

Line 444 creates another restart instance each time:

self.restart_simulation(self.sim_params)

And s, in the forked repo, I just deleted the one self.restart_simulation(self.sim_params) line. Not sure if we want to do it here?

Reply:
Great catch